src/utils/helpers.js

```diff
@@ -16,6 +16,22 @@
 import commonOptions from '../utils/options';
 import { printResult } from '../utils/print';
 
+const regExpAddress = /^\d{1,21}[L|l]$/;
+const regExpAmount = /^\d+(\.\d{1,8})?$/;
+
+export const verifyAddress = (address) => {
+	if (!address.match(regExpAddress)) {
+		throw new Error(`${address} is not a valid address.`);
+	}
+	return true;
+};
+export const verifyAmount = (amount) => {
+	if (!amount.match(regExpAmount)) {
+		throw new Error('Transfer amount must be a number with no more than 8 decimal places.');
```

Comment:
Since this is now in a helper function that's potentially reused in other types of transaction we should leave `Transfer` out.